algorithms-and-data-structures/basic-javascript/index030.js

```diff
@@ -0,0 +1,5 @@
+var xp = 0;
+var health = 100;
+var gold = 50;
+
+// Create a variable to store the current weapon and set it to 0. When a name has two words, the convention is to use so-called "lower camel case". The variable name should look like this: currentWeapon.
```

Comment:
When writing "lower camel case" it is better to actually use lowerCamelCase so the reader has the immediate visual understanding. Also new line each sentence.
```suggestion
// Create a variable to store the current weapon and set it to 0.
// When a name has two words, the convention is to use so-called "lowerCamelCase".
// The variable name should look like this: currentWeapon.
```